client/app/helpers/format-date.coffee

```diff
@@ -1,10 +1,11 @@
 `import Ember from 'ember'`
 
-formatDate = (date, options) ->
+formatDate = (date, format) ->
```

Comment:
yep right and right :) It's been updated to `{{formatDate blahDate format='LLL'}}` and inside the helper it's checking for null like `options.hash.format || 'LL'`